src/install.rst

```diff
@@ -85,6 +85,16 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
             .. group-tab:: Native
 
+               .. warning::
+
+                  If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), first run these commands to ensure Conda creates the environment with ``osx-64`` emulation:
+
+                  .. code-block:: bash
+
+                     CONDA_SUBDIR=osx-64 conda create -n base_osx-64 python   # create a new environment called base_osx-64 with intel packages.
+                     conda activate base_osx-64
+                     conda config --env --set subdir osx-64  # make sure that conda commands in this environment use intel packages
```

Comment:
_non-blocking_

+1 for the comments, but they're practically hidden, even at the widest width:

![image](https://user-images.githubusercontent.com/79913/170599857-3499b3b8-7fbb-472f-a003-0481077273a6.png)

I would probably move them up above each command, e.g.

```suggestion
                     # Create a new environment using Intel packages called base_osx-64
                     CONDA_SUBDIR=osx-64 conda create -n base_osx-64 python

                     # Activate new Intel-based environment
                     conda activate base_osx-64

                     # Ensure future Conda commands in this environment use
                     # Intel packages too.
                     conda config --env --set subdir osx-64
```

Though I haven't looked at how it renders…

Reply:
Good suggestion. I'm not gonna hide that this was straight up copied from https://github.com/conda-forge/miniforge/issues/165#issuecomment-860233092 😅

I chose to keep the last comment on a single line since it renders fine:

![warning-preview](https://user-images.githubusercontent.com/13424970/170772217-b5d4eedb-c728-4362-8853-ae8716f6e38c.png)